packages/surveys/src/components/questions/picture-selection-question.tsx

```
@@ -96,153 +96,150 @@ export function PictureSelectionQuestion({
+                      stroke="currentColor"
+                      strokeWidth="1"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      aria-hidden="true"
+                      className="lucide lucide-image-down-icon lucide-image-down">
+                      <path d="M10.3 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14a2 2 0 0 1 2 2v10l-3.1-3.1a2 2 0 0 0-2.814.014L6 21" />
+                      <path d="m14 19 3 3v-5.5" />
+                      <path d="m17 22 3-3" />
+                      <circle cx="9" cy="9" r="2" />
+                    </svg>
+                  </a>
```

Biome (2.1.2)

[error] Provide screen reader accessible content when using `a` elements.

All links on a page should have content that is accessible to screen readers.
Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies.
Follow these links for more information,
 WCAG 2.4.4
 WCAG 4.1.2

(lint/a11y/useAnchorContent)